Forge.TreeWalker/contracts/ForgeTree.cs

```diff
@@ -217,9 +225,13 @@ public enum RetryPolicyType
         /// Start with MinBackoffMs, then wait Math.Min(MinBackoffMs * 2^(retryCount), MaxBackoffMs).
         /// </summary>
         [EnumMember]
-        ExponentialBackoff = 2
-
-        // TODO: Add a FixedCount type that will give the full timeout duration for the set number of retries.
+        ExponentialBackoff = 2,
+        
+        /// <summary>
+        /// Specify the number of retries for an action.
```

Comment:
>Specify the number of retries for an action. [](start = 12, length = 44)

Replace with this:
Retry a fixed number of times based on RetryPolicy.MaxRetryCount.
Wait RetryPolicy.MinBackoffMs between retries.
Note that Timeout values can be used with this retry type as well.